docs/src/configuration/app/troubleshoot-mounts.md

```diff
@@ -58,43 +86,23 @@ mounts:
         source_path: 'myhiddenfolder'
 ```
 
-## Set up overlapping mount paths
+## Disk space issues
 
-While not recommended, it's possible to setup multiple mount points whose source paths overlap.
-Consider the following example:
+If you are worried about how much disk your mounts are using, check the size with the following command:
 
-```yaml
-mounts:
-    'private':
-        source: local
-        source_path: stuff
-    'secret':
-        source: local
-        source_path:  stuff/secret
+```bash
+$ platform mount:size
 ```
 
-In this configuration, there are two mount points as seen from the application: `~/private` and `~/secret`.
-However, the `secret` mount points to a directory that's also under the mount point for `private`.
-So the `secret` path and the `private/secret` path is the exact same directory.
-
-Although this configuration doesn't cause any technical issues, it may be quite confusing so is generally not recommended.
-
-## Check the size of mounts
-
-You can use standard commands such as `df -ah` to find the total disk usage of mounts
```

Comment:
:pick: Why did we remove the `df` and `du` commands?
Indeed the CLI is enough for most use cases, but df and the likes are useful for more advanced troubleshooting.

Reply:
The CLI uses those commands behind the scenes and it didn't seem to be important to expose that.

Also, if they are "standard commands", then I'm not sure they need to be documented in our docs. I feel like we should focus mostly on documenting what is unique to Platform.

We can add them back if there are specific scenarios they help with, but they just seemed to be a less transparent way of accomplishing the same thing. So I stuck with the one command to simplify it.